backend/src/email/services/email-template.service.ts

```diff
@@ -170,13 +170,17 @@ const storeTemplate = async ({
 }: StoreTemplateInput): Promise<StoreTemplateOutput> => {
   // extract params from template, save to db (this will be done with hook)
   const sanitizedSubject = client.replaceNewLinesAndSanitize(subject)
-  const sanitizedBody = client.filterXSS(body)
+  let sanitizedBody = client.filterXSS(body)
   if (!sanitizedSubject || !sanitizedBody) {
     throw new TemplateError(
       'Message template is invalid as it only contains invalid HTML tags!'
     )
   }
 
+  // Add whitespace, if not present, after email salutation
+  // to prevent phisy looking email preview
+  sanitizedBody = sanitizedBody.replace(/(^.+?,)([^\s].+?<\/p>.*)/, '$1 $2')
```

Comment:
This regex seems to have an issue if the next block after email salutation is not a `p` tag. See [here](https://regex101.com/r/J2YEpo/1) for example.